docs/geoips_overview.rst

```diff
@@ -10,21 +10,21 @@
  | # # # for more details. If you did not receive the license, for more information see:
  | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-GeoIPS Overview
+GeoIPS 2.0 Overview
```

Comment:
Did the 2.0 accidentally squeak in here? I think this is the only file where we reference "GeoIPS" as "GeoIPS 2.0", should probably just keep it as "GeoIPS"?